urls.py

```diff
@@ -41,74 +40,74 @@
+    re_path(r'^$', public.views.feeds, name='feeds-list'),
+    re_path(r'^news/$', cache_page(311)(NewsFeed())),
+    re_path(r'^packages/$', cache_page(313)(PackageFeed())),
+    re_path(r'^packages/(added|removed)/$', cache_page(313)(PackageUpdatesFeed())),
+    re_path(r'^packages/(added|removed)/(?P<arch>[A-z0-9]+)/$', cache_page(313)(PackageUpdatesFeed())),
+    re_path(r'^packages/(added|removed)/all/(?P<repo>[A-z0-9\-]+)/$', cache_page(313)(PackageUpdatesFeed())),
+    re_path(r'^packages/(added|removed)/(?P<arch>[A-z0-9]+)/(?P<repo>[A-z0-9\-]+)/$',
+            cache_page(313)(PackageUpdatesFeed())),
+    re_path(r'^packages/(?P<arch>[A-z0-9]+)/$', cache_page(313)(PackageFeed())),
+    re_path(r'^packages/all/(?P<repo>[A-z0-9\-]+)/$', cache_page(313)(PackageFeed())),
+    re_path(r'^packages/(?P<arch>[A-z0-9]+)/(?P<repo>[A-z0-9\-]+)/$', cache_page(313)(PackageFeed())),
+    re_path(r'^releases/$', cache_page(317)(ReleaseFeed())),
+    re_path(r'^planet/$', cache_page(317)(PlanetFeed()), name='planet-feed'),
 ]
 
 # Old planet.archlinux.org redirects, to be removed once people have migrated.
 urlpatterns.extend([
-    url(r'^planet/rss20.xml$', cache_page(317)(PlanetFeed())),
-    url(r'^planet/atom.xml$', cache_page(317)(PlanetFeed())),
+    re_path(r'^planet/rss20.xml$', cache_page(317)(PlanetFeed())),
+    re_path(r'^planet/atom.xml$', cache_page(317)(PlanetFeed())),
 ])
 
 # Includes and other remaining stuff
 urlpatterns.extend([
-    url(r'^admin/', admin.site.urls),
-    url(r'^devel/', include(devel.urls)),
-    url(r'^feeds/', include(feeds_patterns)),
-    url(r'^groups/', include(packages.urls_groups)),
-    url(r'^mirrorlist/', include(mirrors.urls_mirrorlist)),
-    url(r'^mirrors/', include(mirrors.urls)),
-    url(r'^news/', include(news.urls)),
-    url(r'^packages/', include(packages.urls)),
-    url(r'^releng/', include(releng.urls)),
-    url(r'^todo/', include(todolists.urls)),
-    url(r'^visualize/', include(visualize.urls)),
-    url(r'^opensearch/packages/$', packages.views.opensearch, name='opensearch-packages'),
-    url(r'^opensearch/packages/suggest$', packages.views.opensearch_suggest, name='opensearch-packages-suggest'),
+    re_path(r'^admin/', admin.site.urls),
+    re_path(r'^devel/', include(devel.urls)),
+    re_path(r'^feeds/', include(feeds_patterns)),
+    re_path(r'^groups/', include(packages.urls_groups)),
+    re_path(r'^mirrorlist/', include(mirrors.urls_mirrorlist)),
+    re_path(r'^mirrors/', include(mirrors.urls)),
+    re_path(r'^news/', include(news.urls)),
+    re_path(r'^packages/', include(packages.urls)),
+    re_path(r'^releng/', include(releng.urls)),
+    re_path(r'^todo/', include(todolists.urls)),
+    re_path(r'^visualize/', include(visualize.urls)),
+    re_path(r'^opensearch/packages/$', packages.views.opensearch, name='opensearch-packages'),
+    re_path(r'^opensearch/packages/suggest$', packages.views.opensearch_suggest, name='opensearch-packages-suggest'),
```

Comment:
most of these urls can be converted to a simple `path('admin/') for example.

Reply:
Hi @jelly , yes using `path` for such scenarios makes sense. Getting those changes ready. Thanks!.